src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespacebefore/InputNoWhitespaceBeforeLabel.java

```diff
@@ -0,0 +1,14 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.nowhitespacebefore;
+
+class InputNoWhitespaceBeforeLabel {
+
+	{
+		label1 : // violation
```

Comment:
can you indent this file with spaces instead of tabs?

Reply:
Done.

Reply:
> code change looks good to me, but the issue #8124 needs to be approved

I understand.